### ISSUE TITLE
Proposal: Make Post/Reply Button More Reachable

### ISSUE THREAD

**Issue by machinaeZER0** [thunder-app/thunder#1309]
### Is Your Feature Request Related to a Problem?

I love our post/comment/reply functionality (even moreso with some of the incoming updates to displaying quoted text), but one place I think we could improve is shifting the send/post icon (the little paper airplane/arrow) lower on the screen. Obviously our app is styled differently, but I think the way that most text messaging apps have their send button closer to the bottom makes for a more positive ergonomic experience.

### Feature Description

Since we have such a robust formatting bar for text entry, I don't know that there's a natural place to add a send icon as the app is currently laid out. That said, one idea would be to take the preview icon (the eyeball) and make that the send button, where a long press would give the option to preview instead? This feels rather elegant to me, especially as a person who doesn't often use the preview button, but sees both actions as similar enough that grouping them in a high-profile button "feels good." 

I'd be interested to hear other people's opinions on this - might be a non-issue, but figured it was worth at least bringing up :)

### Additional Context

If we did go the proposed route in the future, just noting that we'd want to make sure that the updated button made it obvious that long-pressing would provide additional options!

**Comment by machinaeZER0** [comment]
@micahmo it sounded like you may have actually mocked up something along these lines at some point?

**Comment by micahmo** [comment]
I think this issue has some overlap with #746 and #788. I will try to dig up my mockup and see if it's still useful! 😆

**Comment by micahmo** [comment]
Ok I found my mockup! My idea was to address the concern of @notBradPitt in [this comment](https://github.com/thunder-app/thunder/pull/455#issuecomment-1648899943) where it was rightly mentioned that, if the button were moved lower, it would be very easy to hit by accident. (And I think that's especially true now that people have muscle memory associated with the preview button down there.) My idea was to show the send button at the bottom only once you've entered preview mode. The benefit is that you won't hit send by accident, but the downside is that you might not always want to preview first, which both @machinaeZER0 and @noodlejetski mentioned.

Anyway, here's the demo. Let me know if this is worth considering at all, or if we should start over. 😆

https://github.com/thunder-app/thunder/assets/7417301/0934147b-ea3d-426c-9aa9-c82d250ac376

**Comment by noodlejetski** [comment]
another downside is that due to the keyboard collapsing on preview, the buttons move around which makes them more difficult to hit. sending a comment that way requires making to very deliberate presses, which makes you hunt for the button twice.

**Comment by machinaeZER0** [comment]
Yeah, my idea would be that Post becomes the main action, with Preview becoming the secondary action on long-press. I understand that I'm saying that as a person who literally never uses the preview button though, which colors my experience! 

Doing it like it is in the demo is pretty slick, and is honestly a pretty good "gut check" that what you're posting is going to look the way you expect, but I would agree that it's probably more presses than people would like to have to sit through every time (especially if they're like me and don't just "rip it" already, haha)

@noodlejetski what are your thoughts on the long press idea, if it were made obvious that the button could be long-pressed?

**Comment by micahmo** [comment]
It sounds like we'll need to brainstorm a bit here to come up with something that everyone likes. 😊 Or this may end up needing to be configurable via settings.

For inspiration, I checked out how some other apps do it.
* Eternity: Preview and send button are at top
* Summit: Send is at top; couldn't find preview
* Sync: Send is at top; couldn't find preview
* Boost: Preview is at bottom (left) and send is at top
* Liftoff: Send is at top; couldn't find preview
* Jerboa: Preview is at bottom (left) and send is at top

All that to say, whatever we come up with will have to be unique. 😊

**Comment by machinaeZER0** [comment]
Welp, hahah. I guess there's a pretty compelling reason to keep it at the top 😂 

Would it be weird to move post/preview functionality into the FAB? That would be pretty consistent with the rest of the app experience...
